src/api/spec/controllers/staging_project/staged_requests_controller_spec.rb

```diff
@@ -2,28 +2,32 @@
 
 RSpec.describe StagingProject::StagedRequestsController, type: :controller do
   render_views
-  describe 'GET #index' do
-    let(:user) { create(:confirmed_user) }
-    let(:project) { create(:project_with_package) }
-    let(:staging_workflow) { create(:staging_workflow_with_staging_projects, project: project) }
-    let(:staging_project) { staging_workflow.staging_projects.first }
-    let(:source_project) { create(:project, name: 'source_project') }
-    let(:target_package) { create(:package, name: 'target_package', project: project) }
-    let(:source_package) { create(:package, name: 'source_package', project: source_project) }
-    let(:bs_request) do
-      create(:bs_request_with_submit_action,
-             target_project: project.name,
-             target_package: target_package.name,
-             source_project: source_project.name,
-             source_package: source_package.name,
-             description: 'BsRequest 1')
-    end
+  include ActiveJob::TestHelper
 
+  let(:other_user) { create(:confirmed_user) }
```

Comment:
[nitpick] what about `user1` and `user2` instead `other_user` and `user`? If you want to leave like that at least define `user` before than `other_user` please